### PR TITLE
nge trash button

### PR DESCRIPTION
**Pull request by smyzin**
- changed the name of the button
- simplify it a little
- created a new Text component for the button
- recheck if the button actually disconnect the wallet

### CODE REVIEW

**Review by MixailE — APPROVED** [review]

File: src/components/ui/Button.tsx [hunk 1]
```diff
@@ -62,11 +62,13 @@ const commonClasses = ({
     cursor({ 'cursor-not-allowed': !available }),
     outlineStyle('focus:outline-none'),
     opacity({ 'opacity-50': !available }),
-    boxShadow({
-      'shadow-2xl': available,
-      'hover:shadow-lg': available,
-      'active:shadow-button-active': available && !gradientFont,
-    }),
+    type !== 'tertiary'
```
Comment (T-Damer) [inline src/components/ui/Button.tsx:65]: Dont do `!={`, do `===`

File: src/components/ui/Text.tsx [hunk 2]
```diff
@@ -488,3 +488,17 @@ export function HighlightedText({
 }) {
   return <div className={highlightedText(bold, center)}>{children}</div>
 }
+
+const trashButtonText = classnames(
+  textColor(
+    'text-secondary',
+    'hover:text-secondary-light-active-transparent',
```
Comment (T-Damer) [inline src/components/ui/Text.tsx:495]: What about mobile hover? After the button is pressed the hover may remain

**Review by T-Damer — APPROVED** [review]

File: src/components/ui/Text.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/ui/Text.tsx]: Fixed

File: src/components/ui/Button.tsx (hunk 1 above)
Comment (smyzin) [reply src/components/ui/Button.tsx]: Done

**Review by T-Damer — APPROVED** [review]

**Pull request merged**